### PR TITLE
Add: copy buttons for id and sid in poke workspace page

### PR DESCRIPTION
**Pull request by Fraggle**
## Description

Add id and sid in the workspace poke page with copy to clipboard buttons as a convenience.

image

## Risk

None

## Deploy Plan

Deploy `front`

### CODE REVIEW

File: front/components/poke/workspace/table.tsx [hunk 1]
```diff
@@ -7,6 +9,33 @@ import {
   PokeTableRow,
 } from "@app/components/poke/shadcn/ui/table";
 
+export function useCopyToClipboard(
```
Comment (flvndvd) [inline front/components/poke/workspace/table.tsx:12]: This guy already exist here:

https://github.com/dust-tt/dust/blob/c39e02f06cdc9dc99176c442e7d76b6d7452055a/front/components/misc/ContentBlockWrapper.tsx#L122

File: front/components/poke/workspace/table.tsx [hunk 2]
```diff
@@ -25,15 +57,69 @@ export function WorkspaceInfoTable({
...
+                />
+              </PokeTableCell>
+            </PokeTableRow>
+            <PokeTableRow>
+              <PokeTableCell>Creation</PokeTableCell>
               <PokeTableCell>{worspaceCreationDay}</PokeTableCell>
             </PokeTableRow>
             <PokeTableRow>
-              <PokeTableCell>SSO Enforced</PokeTableCell>
+              <PokeTableCell>SSO</PokeTableCell>
```
Comment (flvndvd) [inline front/components/poke/workspace/table.tsx:118]: Why removing enforced? SSO can be enabled but not enforced.

**Review by flvndvd — APPROVED** [review]

File: front/components/poke/workspace/table.tsx (hunk 1 above)
Comment (Fraggle) [reply front/components/poke/workspace/table.tsx:12]: Thanks, i overlooked it while searching (i thought it was in sparkle only).
Will move to a common place and use it.

File: front/components/poke/workspace/table.tsx (hunk 2 above)
Comment (Fraggle) [reply front/components/poke/workspace/table.tsx:118]: I wanted to shorten but I didn't know that it could be enabled and not enforced, fixing.

**Pull request merged**